homeassistant/components/imeon_inverter/sensor.py

```diff
@@ -0,0 +1,79 @@
+"""Imeon inverter sensor support."""
+
+import logging
+
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import InverterCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Create each sensor for a given config entry."""
+
+    # Get Inverter from UUID
+    IC: InverterCoordinator = entry.runtime_data
+
+    async_add_entities(
+        ([InverterSensor(IC, "meter_power", entry, "Meter Power", "W")]), True
```

Comment:
will there be more sensors

Reply:
sure, i just set one but i will use a dict to refer to every sensors

Reply:
we normally use `EntityDescriptions` for multiple sensors, so maybe give that a look then